operators/mongodb-atlas-kubernetes/1.5.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: mongodb-atlas-kubernetes
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.15.0+git
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v2
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
+
+  com.redhat.openshift.versions: "v4.8"
```

Comment:
```suggestion
  com.redhat.openshift.versions: "v4.8-v4.11"
```

Reply:
This version is using deprecated api that is missing in v4.12 (k8s v1.25). You can limit your operator version to be present in range `v4.8-v4.11`